src/go/rpk/pkg/cli/cmd/cluster/license/set.go

```diff
@@ -0,0 +1,71 @@
+package license
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newSetCommand(fs afero.Fs) *cobra.Command {
+	var licPath string
+	cmd := &cobra.Command{
+		Use:   "set",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Upload license to the cluster",
+		Long: `Upload license to the cluster
+
+You can either provide a path to a file containing the license:
+
+    rpk cluster license set --path /home/organization/redpanda.license
+
+Or inline the license string:
+
+    rpk cluster license set <license string>
+
+If neither of them is present, rpk will look for the license in the
+default location '/etc/redpanda/redpanda.license'
+`,
+
+		Run: func(cmd *cobra.Command, args []string) {
+			if licPath != "" && len(args) > 0 {
+				out.Die("inline license can't be passed if flag '--path' is set")
+			}
+			if licPath == "" && len(args) == 0 {
+				fmt.Println("Neither license file nor inline license was provided, looking for license file in '/etc/redpanda/redpanda.license'")
```

Comment:
How about `looking for license file in` => `checking`, to make the line shorter?
also suffix with period

Reply:
Actually, why not make `/etc/redpanda/redpanda.license` the default value for the flag, and if both are empty here, exit with an error?